### DISCUSSION TITLE
workload or workload detail without  flex group

### DISCUSSION THREAD

**Discussion by leejshades** [NetApp/harvest#3050] [Q&A]
may i show workload or workload detail with flex group？

**Comment by rahulguptajss** [comment]
@leejshades The Workload dashboard in Harvest displays both FlexVol and FlexGroup data. Are you requesting a filter that allows you to select only FlexGroups in the relevant dashboard? Could you please provide more details about your request and specify which dashboard you are referring to?

**Comment by leejshades** [comment]
got it ,

**Comment by rahulguptajss** [comment]
@leejshades Have you found the information you needed? If no further changes are required, can we close this discussion?

**Reply by leejshades** [reply]
> @leejshades Have you found the information you needed? If no further changes are required, can we close this discussion?

no , thanks
